index.html

```diff
@@ -0,0 +1,190 @@
+                        <li><a href="">Products</a></li>
+                        <li><a href="">Portfolio</a></li>
+                        <li><a href="">Contact us</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        <div>
+            <a href="#">
+                <img src="img/search.png" alt="search">
+            </a>
+        </div>
+        <div>     
+            <a href="#">
+                <img src="img/notification.png" alt="search">
+            </a>
+        </div>
+        <button>
+            <img src="img/Avatar.png" alt="Avatar">
+        </button>
+    </header>
+    <main> 
+        <div>
+            <h1>Contact us</h1>
+            <strong>Need an experienced and skilled hand with custom IT projects?<br>Fill out the form to get a free consultation.</strong>
+        </div>
+        <div>
+            <form>
+                <p>Your company name*</p>
+                <input type="text">
+                <p>Nature of Business*</p>
+                <input type="text">
+                <p>Data of registration</p>
+                <input type="date" name="" id="">
+            </form>
+        </div>
+        <div>
+            <form>
+                <p>Country of registration*</p>
+                <select name="">
+                    <option value="">Select</option>
+                </select>
+                <p>Address</p>
+                <input type="text">
+                <p>Postcode</p>
+                <input type="text">
+            </form>
+        </div>
+        <hr>
+        <div>
+            <form>
+                <p>Contact name</p>
+                <input type="text">
+                <p>Contact phone</p>
+                <input type="tel">
+                <p>Email*</p>
+                <input type="email" name="" id="">
+            </form>
+        </div>
+        <div>
+            <form>
+                <p>Let’s talk about your idea</p>
+                <textarea name="" id="" cols="30" rows="10" maxlength="320"></textarea>
+            </form>
+        </div>
+        <div>
+            <form action="">
+                <input type="range" min="0" max="100"
+                list="tickmarks">
+                <datalist id="tickmarks">
+                <option value="0"></option>
+                <option value="10"></option>
+                <option value="20"></option>
+                <option value="30"></option>
+                <option value="40"></option>
+                <option value="50"></option>
+                <option value="60"></option>
+                <option value="70"></option>
+                <option value="80"></option>
+                <option value="90"></option>
+                <option value="100"></option>
+                </datalist>
+            </form>
+        </div>
+        <hr>
+        <div>
+            <img src="img/upload.png" alt="button_upload">
+            <a href=""> Upload Additional file</a>

```

Comment:
this is also special type of input
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file